databricks.yml

```
@@ -0,0 +1,9 @@
+bundle:
+  name: marvelous-databricks-course-Gijsvoogd
+
+targets:
+  dev:
+    mode: development
+    default: true
+    workspace:
+      host: redacted
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)